petrinet/petrinet_schema.json

```diff
@@ -90,6 +93,15 @@
         },
         "required": ["type", "parameters"],
         "additionalProperties": true
+      },
+      "grounding": {
+        "type": "object",
+        "properties": {
+          "identifiers": {"type": "object"},
+          "context": {"type": "object"}
```

Comment:
@cthoyt what's an example of the context? I think it's fine that it's just an object but it would help to have the `sir.json` example show what _could_ be in that context I think.

Reply:
I showed an example of the format in this comment: https://github.com/DARPA-ASKEM/Model-Representations/pull/5#issuecomment-1516479302. The issue is that the states in simple SIR model don't really have context to be represented on the states, so it might be confusing to add something there.

Reply:
ah got it! missed that